src/components/Menu/entries.js

```diff
@@ -20,117 +20,162 @@
  *
  */
 
+import {
+	Undo,
+	Redo,
+	FormatBold,
+	FormatItalic,
+	FormatUnderline,
+	FormatStrikethrough,
+	FormatHeader1,
+	FormatHeader2,
+	FormatHeader3,
+	FormatHeader4,
+	FormatHeader5,
+	FormatHeader6,
+	FormatListNumbered,
+	FormatListBulleted,
+	FormatListCheckbox,
+	FormatQuote,
+	Info,
+	Positive,
+	Warn,
+	Danger,
+	CodeTags,
+	Table,
+	Emoticon,
+	Help,
+	Images,
+} from '../icons.js'
+import EmojiPickerAction from './EmojiPickerAction.vue'
+import ActionImageUpload from './ActionImageUpload.vue'
+
 export default [
 	{
+		_key: 'undo',
 		label: t('text', 'Undo'),
 		keyChar: 'z',
 		keyModifiers: ['ctrl'],
-		class: 'icon-undo',
+		icon: Undo,
 		action: (command) => command.undo(),
 		priority: 5,
 	},
 	{
+		_key: 'redo',
 		label: t('text', 'Redo'),
 		keyChar: 'y',
 		keyModifiers: ['ctrl'],
-		class: 'icon-redo',
+		icon: Redo,
 		action: (command) => command.redo(),
 		priority: 11,
 	},
 	{
+		_key: 'bold',
 		label: t('text', 'Bold'),
 		keyChar: 'b',
 		keyModifiers: ['ctrl'],
-		class: 'icon-bold',
+		icon: FormatBold,
 		isActive: 'strong',
 		action: (command) => {
 			return command.toggleBold()
 		},
 		priority: 6,
 	},
 	{
+		_key: 'italic',
 		label: t('text', 'Italic'),
 		keyChar: 'i',
 		keyModifiers: ['ctrl'],
-		class: 'icon-italic',
+		icon: FormatItalic,
 		isActive: 'em',
 		action: (command) => {
 			return command.toggleItalic()
 		},
 		priority: 7,
 	},
 	{
+		_key: 'underline',
 		label: t('text', 'Underline'),
 		keyChar: 'u',
 		keyModifiers: ['ctrl'],
-		class: 'icon-underline',
+		icon: FormatUnderline,
 		isActive: 'underline',
 		action: (command) => {
 			return command.toggleUnderline()
 		},
 		priority: 14,
 	},
 	{
+		_key: 'strikethrough',
 		label: t('text', 'Strikethrough'),
 		keyChar: 'd',
 		keyModifiers: ['ctrl'],
-		class: 'icon-strike',
+		icon: FormatStrikethrough,
 		isActive: 'strike',
 		action: (command) => {
 			return command.toggleStrike()
 		},
 		priority: 15,
 	},
 	{
+		_key: 'headings',
 		label: t('text', 'Headings'),
 		keyChar: '1…6',
 		keyModifiers: ['ctrl', 'shift'],
 		visible: false,
+		icon: FormatHeader1,
+		isActive: 'heading',
 		children: [
 			{
+				_key: 'headings-h1',
 				label: t('text', 'Heading 1'),
-				class: 'icon-h1',
+				icon: FormatHeader1,
 				isActive: ['heading', { level: 1 }],
 				action: (command) => {
 					return command.toggleHeading({ level: 1 })
 				},
 			},
 			{
+				_key: 'headings-h2',
 				label: t('text', 'Heading 2'),
-				class: 'icon-h2',
+				icon: FormatHeader2,
 				isActive: ['heading', { level: 2 }],
 				action: (command) => {
 					return command.toggleHeading({ level: 2 })
 				},
 			},
 			{
+				_key: 'headings-h3',
 				label: t('text', 'Heading 3'),
-				class: 'icon-h3',
+				icon: FormatHeader3,
 				isActive: ['heading', { level: 3 }],
 				action: (command) => {
 					return command.toggleHeading({ level: 3 })
 				},
 			},
 			{
+				_key: 'headings-h4',
```

Comment:
Any specific reasoning for the underscore prefix? I wouldn't consider this a kind of private property?

Reply:
Makes sense, I will change.